po/de.po

```diff
@@ -22753,24 +22727,20 @@ msgid "No branch name specified"
 msgstr "Kein Branchname spezifiziert"
 
 #: builtin/stash.c:808
-#, fuzzy
 msgid "failed to parse tree"
-msgstr "Fehler beim Parsen von %s."
+msgstr "Parsen der Tree-Objekte fehlgeschlagen"
 
 #: builtin/stash.c:819
-#, fuzzy
 msgid "failed to unpack trees"
-msgstr "Fehler beim Entpacken des Tree-Objektes von HEAD."
+msgstr "Entpacken der Tree-Objekte fehlgeschlagen"
 
 #: builtin/stash.c:839
-#, fuzzy
 msgid "include untracked files in the stash"
 msgstr "unversionierte Dateien in Stash einbeziehen"
 
 #: builtin/stash.c:842
-#, fuzzy
 msgid "only show untracked files in the stash"
-msgstr "unversionierte Dateien in Stash einbeziehen"
+msgstr "nur unversionierte Dateien in Stash anzeigen"
```

Comment:
im Stash

Reply:
hab gleich auch noch die anderen zwei Referenzen "in Stash" im Deutschen korrigiert.. wir nutzen ja "der Stash" :+1: